MimeMapping.sln

```diff
@@ -6,7 +6,6 @@ MinimumVisualStudioVersion = 10.0.40219.1
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{53C2C92A-AA91-4027-A6A3-AAECEE661B38}"
 	ProjectSection(SolutionItems) = preProject
 		.editorconfig = .editorconfig
-		MimeMapping.nuspec = MimeMapping.nuspec
```

Comment:
Why this change?

Reply:
You forgot to delete this from .sln, it doesn't exist